artemis/reporting/export.py

```diff
@@ -0,0 +1,273 @@
+import dataclasses
+import datetime
+import glob
+import json
+import os
+import urllib.parse
+from collections import Counter, defaultdict
+from typing import Any, DefaultDict, Dict, List, Set
+
+import tqdm
+import typer
+from jinja2 import BaseLoader, Environment, StrictUndefined
+
+from artemis.config import Config
+from artemis.db import DB
+from artemis.json_utils import JSONEncoderAdditionalTypes
+
+from .base import (
+    Language,
+    Report,
+    ReportType,
+    build_message_template,
+    reports_from_task_result,
+)
+from .utils import get_top_level_target_if_present
+
+environment = Environment(loader=BaseLoader(), undefined=StrictUndefined, trim_blocks=True, lstrip_blocks=True)
+OUTPUT_LOCATION = "./output/autoreporter/"
+
+
+def _get_domain_or_ip_only(data: str) -> str:
+    """Extracts domain/ip from URL or host:port string."""
+    if "://" in data:
+        host = urllib.parse.urlparse(data).hostname
+        assert host
+        return host
+    if ":" in data:
+        return data.split(":")[0]
+    return data
```

Comment:
wtf
I understand we don't believe in proper URL parsing. How about `admin:admin@www.cert.pl` ?